bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -751,6 +754,39 @@ namespace JsRTApiTest
         REQUIRE(JsSetException(exception) == JsNoError);
         REQUIRE(JsHasException(&value) == JsNoError);
         CHECK(value == true);
+
+        REQUIRE(JsGetAndClearExceptionWithMetadata(&exceptionMetadata) == JsNoError);
```

Comment:
would be nice to verify this API in the case where there is no exception thrown?